agent/php_user_instrument.c

```diff
@@ -173,10 +173,10 @@ static void reset_wraprec(nruserfn_t* wraprec) {
  * `newrelic.transaction_tracer.custom` 3)
  * nr_php_user_function_add_declared_callback (prior to PHP 7.3) 4) from
  * function `nr_php_wrap_user_function` called from php_wrapper sets the wraprec
- * with framework specific instrumentation. 5) from function
- * `nr_php_wrap_callable` (in `php_wrapper.c`) used only by Wordpress and predis
- * for custom instrumentation that adds `is_transient` wrappers that get cleaned
- * up with each shutdown.
+ * with framework specific instrumentation. Optionally sets `is_transient`.
```

Comment:
1. There is no `is_transient` field in `nruserfn_t` data type any more.
2. Is setting `transience` field optional? The new API requires explicit transience value to be used at the time the wraprec is created.

Reply:
1. reworded in [a52a440](https://github.com/newrelic/newrelic-php-agent/pull/679/commits/a52a440e0fc1aae7e8c3a2a08d38037201596ede)
2. it is optional. There is the regular call to nr_php_wrap_user_function (which sets not_transient) and then the call nr_php_wrap_user_function_with_transience which takes transience as a parameter.